CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# dbt_apple_store_source v0.3.1
```

Comment:
Was this missed by chance?

Reply:
Yeah my bad, just completed all the changelogs